Training/Training/tempOut.txt

```diff
@@ -0,0 +1 @@
+00010000001
```

Comment:
This file should not be committed.  It is going to be generated anyway and is not necessary for the program to work.